spec/factories/likes.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :like do
```

Comment:
Same here, let factory know about your associations:

```
FactoryBot.define do
  factory :likeable do
    message { "Lorem Ipsum Text" }

    association :post 
    association :user
  end
end
```

This will allow FactoryBot to create a Post and User every time a Likeable is created.